lib/trust_is_risk.js

```diff
@@ -0,0 +1,202 @@
+//      
+var bcoin = require('bcoin');
+var assert = require('assert');
+var Address = bcoin.primitives.Address;
+
+                      // base58 bitcoin address
+                    
+                
+             
+                 
+                 
+                                                               
+ 
+                               
+                     
+
+class TrustIsRisk {
+                       
+  
+                 
+                        
+                            
+      
+   
+
+               
+                                 
+   
+
+  constructor(node                 ) {
+    this.node = node;
+    this.directTrust = {};
+    this.TXToTrust = {};
+
+    this.node.on('tx', this.addTX.bind(this));
+  }
+
+  getDirectTrust(from         , to         )          {
+    if (!this.directTrust.hasOwnProperty(from)) return 0;
```

Comment:
Always [use braces](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/) even if your body is one line.